core/src/main/java/io/parsingdata/metal/data/selection/ByType.java

```diff
@@ -16,26 +16,27 @@
 
 package io.parsingdata.metal.data.selection;
 
+import static io.parsingdata.metal.Util.checkNotNull;
+
 import io.parsingdata.metal.data.ParseGraph;
 import io.parsingdata.metal.data.ParseGraphList;
 import io.parsingdata.metal.data.ParseItem;
-
-import static io.parsingdata.metal.Util.checkNotNull;
+import io.parsingdata.metal.data.ParseItemList;
 
 public final class ByType {
 
     private ByType() {}
 
-    public static ParseGraphList getRefs(final ParseGraph graph) {
+    public static ParseItemList getRefs(final ParseGraph graph) {
```

Comment:
Now that this has changed from `ParseGraphList` to `ParseItemList`, there is only one place where we still use `ParseGraphList`: the `ByType.getGraphs()` method. We do not use that method anymore (except in tests), so here is an idea: let's remove `ParseGraphList`. Or is there a use case for `getGraphs()`?

If you agree, let's create a separate issue.